### DISCUSSION TITLE
Is this compatible with Mikrotik RouterOS?

### DISCUSSION THREAD

**Discussion by nzsambo** [akvorado/akvorado#521] [Q&A] [answered]
Before I continue troubleshooting I would like to know if this is compatible, and if anyone has actually successfully got Akvorado to export flows from /ip traffic-flow in mikrotik?

The `sampling rate missing` seems to be what increments 

```
tuna@docker13:~$ curl -s http://192.168.23.113:8081/api/v0/inlet/metrics | grep '^akvorado_inlet' | grep _errors
akvorado_inlet_core_flows_errors{error="SNMP cache miss",exporter="192.168.23.1"} 30
akvorado_inlet_core_flows_errors{error="sampling rate missing",exporter="192.168.23.1"} 97792
akvorado_inlet_flow_decoder_netflow_errors_count{error="template not found",exporter="192.168.23.1"} 7
```

**Comment by vincentbernat** [comment, marked as answer]
Use `inlet`→`core`→`default-sampling-rate` to workaround this.

**Reply by nzsambo** [reply]
Thank you sir, that did the trick.

**Reply by edgenative** [reply]
@nzsambo what does your mikrotik config look like?  even using default-sampling-rate I can't get sensible results?  I get spikes of traffic 3-4x the actual interface size.

**Comment by nzsambo** [comment]
I ended up turning off packet sampling completely and making the default sampling rate 1. I am only monitoring the internal LAN interfaces. Sorry I don't have access to the device from here, it's a very simple config. I will make sure to reply here with it when I canOn 2/03/2023, at 1:08 PM, edgenative ***@***.***> wrote:﻿
@nzsambo what does your mikrotik config look like?  even using default-sampling-rate I can't get sensible results?  I get spikes of traffic 3-4x the actual interface size.

**Reply by HeriAditya** [reply]
can u help me to install and export trafic flow from mikrotik??? I'm ready to pay.. adityaheri@yahoo.com
